src/context/useSavedArmies.tsx

```diff
@@ -54,12 +51,13 @@ const SavedArmiesProvider: React.FC = ({ children }) => {
   const { user } = useAuth0()
   const { subscription, isActive } = useSubscription()
   const [savedArmies, setSavedArmies] = useState<ISavedArmyFromApi[]>([])
-  const [loadedArmy, setLoadedArmy] = useState<TLoadedArmy>(null)
   const [favoriteFaction, setFavoriteFaction] = useState<TSupportedFaction | null>(null)
   const [waitingForApi, setWaitingForApi] = useState(false)
 
```

Comment:
create a function here like so

```tsx
const useSetLoadedArmy = ({id, armyName}) => {
  LocalLoadedArmy.set({ id, armyName })
  setLoadedArmy({ id, armyName })
}
```

And export this from Context instead of `setLoadedArmy`. This means that you'll keep the LocaLStorage and the context state in sync.